ENH: covariates collection (#234)
TheAxonLab/hcph-sops: docs/data-collection/pre-session.md

    ![nuprep-tube](https://shop.neurospec.com/media/catalog/product/cache/8af976a03d45f9b6d7f91e69feeeffb9/w/e/weaver_nuprep-skin-prep-gel-single-tube-front.jpg)

- [ ] Open a new issue in [the github repository](https://github.com/TheAxonLab/hcph-sops/) to collect comments and annotations about the session (a laptop should be available to fill the issue).
    - [ ] Under the section `Issues`, click on <span class="consolebutton green">New issue</span>

>     - [ ] This should lead you a page that looks like this 

Reviewer: ```suggestion
    - [ ] This should lead you to a page that looks like this 
```

    ![nuprep-tube](https://shop.neurospec.com/media/catalog/product/cache/8af976a03d45f9b6d7f91e69feeeffb9/w/e/weaver_nuprep-skin-prep-gel-single-tube-front.jpg)

> - [ ] Open a new issue in [the github repository](https://github.com/TheAxonLab/hcph-sops/) to collect comments and annotations about the session [ON WHICH COMPUTER SHOULD THIS BE DONE?]

Reviewer: Fixed by #219 
```suggestion
- [ ] Open a new issue in [the github repository](https://github.com/TheAxonLab/hcph-sops/) to collect comments and annotations about the session (a laptop should be available to fill the issue).
```

    - [ ] Under the section `Issues`, click on <span class="consolebutton green">New issue</span>
    - [ ] This should lead you a page that looks like this 
        ![](../assets/images/session_issue_template.png)
        - [ ] Click on <span class="consolebutton green">Get started</span> on the issue template `Scan session`.
    - [ ] Modify the title of the issue by replacing `yyy` with the session index. If you don't remember the session index of today, check the session index of the last issue.
- [ ] Verify that your phone is on ringing mode so the participants can reach you
- [ ] Check the time regularly to be on time to meet with the participant at the predefined location

## Collection of covariates

> - [ ] Open a new issue in [the github repository dedicated to collecting covariates](https://github.com/TheAxonLab/hcph-mood-quest/) [ON WHICH COMPUTER SHOULD THIS BE DONE?]

Reviewer: Same, fixed by #219 
```suggestion
- [ ] Open a new issue in [the github repository dedicated to collecting covariates](https://github.com/TheAxonLab/hcph-mood-quest/) (a laptop should be available to fill the issue).
```
